.travis.yml

```diff
@@ -6,6 +6,22 @@
 # please add the following to your local git config:
 #   git config merge.ours.driver true
 ##########
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
 branches:
   only:
```

Comment:
I saw that Spark did this (https://github.com/apache/spark/blob/master/.travis.yml) but if you think it'd be better to exclude this, I'll go ahead and update. Thanks!